spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -473,6 +490,10 @@ private static String parentName(String[] fieldNames) {
     return null;
   }
 
+  public static String describe(List<org.apache.iceberg.expressions.Expression> exprs) {
+    return exprs.stream().map(Spark3Util::describe).collect(Collectors.joining(", "));
```

Comment:
Not a blocker, but should we sanitize the expressions using `ExpressionUtil.sanitize` here? There is a risk of logging IDs if we log this.

Reply:
Actually, this is used by `toString`, so it looks like the right behavior is to not sanitize because this is put into Spark plans where the full filter should be shown. We may want to introduce a sanitized describe later though, for logging purposes.

Reply:
That's exactly my thought process too. I wanted to sanitize first but then realized it is used in the plan and kept the existing behavior.